src/components/Payment/LegalCopy/LegalCopy.js

```diff
@@ -18,7 +19,7 @@ const LegalCopy = () => {
   return (
     <LegalNoteWrapperStyled>
       <LegalTextStyled marginBottom="24px">
-        <Trans i18nKey="legal-notes-acknowledge">
+        <Trans i18nKey="legal-notes-acknowledge" t={t}>
```

Comment:
do we need this 't={t}'?

Reply:
Now I see it is needed 😅 will revert it back